pkg/goversion/go12.go

```diff
@@ -0,0 +1,21 @@
+// +build go1.12
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package goversion
+
+const requiresGo1dot12 = true
```

Comment:
Perhaps call this variable could be renamed to `wasCompiledWithGo1Dot12OrHigher`.